### PR TITLE
SDK v3.2

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

File: pages/hub/releases/_meta.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "15-sdk-v3.2": "04/15 SDK v3.2",
```
Comment (on47sky) [inline pages/hub/releases/_meta.json:2]: 16-sdk-v3.2 :)

**Review by on47sky — APPROVED** [review]

File: pages/hub/releases/_meta.json (hunk 1 above)
Comment (CleanBread) [reply pages/hub/releases/_meta.json]: thanks)
a87284e83b6407822a2a4f289753436f5e8f3fdd

File: pages/hub/apps/sdk/write-hooks/useDeBridgeBet.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
+import { Callout } from 'components';
+
+This hook is used for users to place bets from others blockchains. For instance, if a user desires to wager on Polygon but lacks tokens for it, they can utilize tokens from Arbitrum instead.
```
Comment (Avvrik) [inline pages/hub/apps/sdk/write-hooks/useDeBridgeBet.mdx:3]: This hook enables users to place bets using tokens from different blockchains. For example, if someone wants to bet on Polygon but does not have the necessary tokens, they can opt to use tokens from the Arbitrum (or any other supported network) instead.

**Review by Avvrik — COMMENTED** [review]
reformulated one paragraph

File: pages/hub/apps/sdk/write-hooks/useDeBridgeBet.mdx (hunk 2 above)
Comment (CleanBread) [reply pages/hub/apps/sdk/write-hooks/useDeBridgeBet.mdx]: 7983ca0f7e6ba2f1727dfab481f4ca41c4bede58

**Pull request merged**